entries/l/lhv.ee.json

```diff
@@ -2,12 +2,10 @@
   "LHV": {
     "domain": "lhv.ee",
     "url": "https://www.lhv.ee/",
-    "contact": {
-      "email": "info@lhv.ee",
-      "facebook": "LHVPank",
-      "twitter": "ellhaavee",
-      "language": "et"
-    },
+    "tfa": [
+      "custom-software",
+      "custom-hardware"
```

Comment:
`custom-software` and `custom-hardware` tags are needed since they have been included in the `tfa` array.